README.md

```diff
@@ -336,6 +336,19 @@ Redshift cluster and/or don't have requirements to keep the table availability h
     </td>
  </tr>
  <tr>
+    <td><tt>preactions</tt></td>
+    <td>No</td>
+    <td>No default</td>
+    <td>
+<p>This can be a <tt>;</tt> separated list of SQL commands to be executed before loading <tt>COPY</tt> command.
```

Comment:
I realized that the text here (and in `postactions`) is potentially ambiguous and it might be worth editing the docs to clarify that pre- and post-actions are only run when writing data _to_ Redshift. This is probably obvious, though.